js/index.js

```diff
@@ -0,0 +1,97 @@
+function shuffle(array) {
+    let currentIndex = array.length, randomIndex;
+
+    while (currentIndex !== 0) {
+
+        randomIndex = Math.floor(Math.random() * currentIndex);
+        currentIndex--;
+
+        [array[currentIndex], array[randomIndex]] = [
+            array[randomIndex], array[currentIndex]];
+    }
+
+    return array;
+}
+
+fetch('words.json')
+    .then(response => response.json())
+    .then(words => {
+        const wordsArr = words['fiveLetterWords']
+        const shuffledWords = shuffle(wordsArr)
+        let expectedWord = shuffledWords[0].toUpperCase().split("")
+        console.log(expectedWord)
+        let attemptedWord = []
+        let characterCounter = 0
+// let attemptCounter = 0 - useful later
+
+// MAKE ON SCREEN LETTERS WORK
+        const targetOnScreenLetters = document.querySelectorAll('.key')
+        const targetBackspace = document.querySelector('.backspace')
+        const targetEnter = document.querySelector('.enter')
+        targetOnScreenLetters.forEach(function (letter) {
+            letter.addEventListener('click', function () {
+                if (attemptedWord.length < 5) {
+                    attemptedWord.push(letter.innerHTML)
+                    document.querySelector('.tile' + characterCounter).innerHTML = letter.innerHTML
+                    characterCounter++
+                    console.log(attemptedWord, characterCounter)
+                }
+            })
+        })
+
+        targetBackspace.addEventListener('click', deleteLetter)
+        targetEnter.addEventListener('click', enterPressed)
+
+        function deleteLetter () {
+            if (characterCounter > 0) {
+                attemptedWord = attemptedWord.slice(0, attemptedWord.length - 1)
+                characterCounter--
+                document.querySelector('.tile' + characterCounter).innerHTML = ''
+                // characterCounter--
+                console.log(attemptedWord, characterCounter)
+            }
+        }
+
+        function enterPressed () {
+            if (attemptedWord.length === 5) {
+                const result = checkWord(expectedWord,attemptedWord)
+                alert(result)
+            }
+        }
+
+// MAKE THE REAL KEYBOARD WORK
+        const characterSet = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+        document.addEventListener('keyup', (event) => {
+            if (attemptedWord.length < 5) {
+                if (characterSet.includes(event.key)) {
```

Comment:
Rather than nesting two if statements here, I would suggest using a single if with && for both conditions. 

I would only have them nested like this if you wanted separate else's on both conditions

Reply:
done